route/auth.js

```diff
@@ -0,0 +1,61 @@
+const Router   = require('express').Router;
+const router   = new Router();
+const request  = require('request');
+
+
+function authenticate(req, res, next) {
+  req.logger.verbose('Authenticating user through Auth0');
+
+  const username = req.body.username;
+  const password = req.body.password;
+
+  if (!username || !password) {
+    return res.send(400, 'Missing username and/or password parameters.').end();
```

Comment:
I think in express 4 is `res.status(400).send('Missing username and/or password parameters.');`